packages/editor/src/client/atoms/toolbar.tsx

```diff
@@ -166,20 +162,31 @@ export const SubMenuButton = forwardRef<PopoverProps, SubMenuButtonProps>(
 )
 
 export function ToolbarDivider() {
-  return (
-    <div
-      style={{
-        width: '1px',
-        alignSelf: 'stretch',
+  return <Divider style={{height: '1.5em'}} vertical />
+}
 
-        marginLeft: 20,
-        marginRight: 20,
-        marginTop: 2,
-        marginBottom: 2,
+interface ControlsContainerProps {
+  children: ReactNode
+  dividerTop?: boolean
+  dividerBottom?: boolean
+}
 
-        backgroundColor: 'gray'
-      }}
-    />
+export function ControlsContainer({children, dividerTop, dividerBottom}: ControlsContainerProps) {
+  // TODO find rsuite component
```

Comment:
Do you want to create a ticket for this?

Reply:
Jep, think this would also be a nice first issue ?

Reply:
👍🏾